src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.Analyzers.targets

```
@@ -50,9 +50,15 @@ Copyright (c) .NET Foundation. All rights reserved.
                                     $([MSBuild]::VersionGreaterThanOrEquals($(EffectiveAnalysisLevel), '5.0'))">true</EnableNETAnalyzers>
     <EnableNETAnalyzers Condition="'$(EnableNETAnalyzers)' == ''">false</EnableNETAnalyzers>
 
+    <!-- Intermediate step to enable ILLink.Analyzers so ILLink, Blazor, Xamarin, AOT, etc. can enable the same flags -->
+    <EnableSingleFileAnalyzer Condition="'$(EnableSingleFileAnalyzer)' == '' And
+                              '$(PublishSingleFile)' == 'true'">true</EnableSingleFileAnalyzer>
```

formatting nit - could you align the starting `'` in the two lines of the condition, like we do above?
```suggestion
                                         '$(PublishSingleFile)' == 'true'">true</EnableSingleFileAnalyzer>
```